src/components/VideoDetail/VideoDetail.component.jsx

```diff
@@ -1,20 +1,35 @@
 import React, { useState, useEffect } from 'react';
 import { VideoFrame, UnorderedList, LinkVideo } from './styled';
 import Grid from '@material-ui/core/Grid';
-
+import { storage } from '../../utils/storage';
 import { youtube } from '../../api/youtube';
 
-const VideoDetail = ({ videoId, videoName }) => {
+const VideoDetail = ({ videoId, videoName, favorite }) => {
+  favorite = favorite ?? 'false';
   const [related, setRelated] = useState([]);
+
+  const path = favorite === 'true' ? 'favoritedetails' : 'details';
```

Comment:
if `favorite` were of boolean type it would be easier to handle this line
```suggestion
  const path = favorite ? 'favoritedetails' : 'details';
```